src/classes/GE_FormRendererService.cls

```diff
@@ -218,12 +218,26 @@ public with sharing class GE_FormRendererService {
         return null;
     }
 
+    @AuraEnabled(Cacheable=true)
+    public static List<Allocation__c> getDonationAllocations(Id donationId) {
+        if(donationId == null) {
+            return null;
```

Comment:
I don't think this causes any issues, but is this a common practice for a method that returns a list to return null? Should we return an empty list instead?

Reply:
fixed